worklets/Overview.bs

```diff
@@ -336,8 +342,8 @@ the user agent <em>must</em> run the following steps:
             1. Let |insideSettings| be the {{WorkletGlobalScope}}'s associated <a>environment
                 settings object</a>.
 
-            2. <a>Fetch a module worklet script tree</a> given |resolvedModuleURL|,
-                |outsideSettings|, and |settingsObject|.
+            2. <a>Fetch a module worklet script tree</a> given the current {{Worklet}},
```

Comment:
Nit: "the current Worklet" is not defined here. I think you want to add a step at the top of this method definition that does "Let _worklet_ be this `Worklet` object" or similar. (Other acceptable phrasing: "the `Worklet` object on which this method was called".) Then you can use _worklet_ later.